### PR TITLE
Merge fenix-quarantine

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -282,7 +282,7 @@
     <!-- Screen title for adding a bookmarks folder -->
     <string name="bookmark_add_folder">Add folder</string>
     <!-- Snackbar title shown after a bookmark has been created. -->
-    <string name="bookmark_created_snackbar">Bookmark created.</string>
+    <string name="bookmark_created_snackbar">Bookmark saved!</string>
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: This needs a new string ID

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -300,4 +313,80 @@
...
+        to give users access to site specific information / settings. For example:
+        Secure settings status and a button to modify site permissions -->
+    <string name="quick_settings_sheet">Quick settings sheet</string>
+    <!-- Label that indicates that this option it the recommended one -->
+    <string name="phone_feature_recommended">Recommended</string>
+    <!-- button that allows editing site permissions settings -->
+    <string name="quick_settings_sheet_manage_site_permissions">Manage site permissions</string>
+    <!-- Button label for clear all the information of a site permissions-->
+    <string name="clear_permissions">Clear permissions</string>
+    <!-- Button label for clear a site permissions-->
```
[the remark this answers is not in the document]
Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: SOrry this should be about the <string name="clear_permission"> string. Can't delete my comment

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -281,8 +292,10 @@
     <string name="bookmark_select_folder">Select folder</string>
     <!-- Screen title for adding a bookmarks folder -->
     <string name="bookmark_add_folder">Add folder</string>
-    <!-- Snackbar title shown after a bookmark has been created. -->
+    <!-- deprecated: Snackbar title shown after a bookmark has been created. -->
```
[the remark this answers is not in the document]
Comment (Pike) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: It's not helping, we can just remove the old string.

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -300,4 +313,80 @@
...
+    <string name="quick_settings_sheet">Quick settings sheet</string>
+    <!-- Label that indicates that this option it the recommended one -->
+    <string name="phone_feature_recommended">Recommended</string>
+    <!-- button that allows editing site permissions settings -->
+    <string name="quick_settings_sheet_manage_site_permissions">Manage site permissions</string>
+    <!-- Button label for clear all the information of a site permissions-->
+    <string name="clear_permissions">Clear permissions</string>
+    <!-- Button label for clear a site permissions-->
+    <string name="clear_permission">Clear permission</string>
+    <!-- Button label for clear all the information fall all sites-->
```
[the remark this answers is not in the document]
Comment (Pike) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: "for all", I guess.

**Comment by Delphine** [comment]
I think all the fixes have been addressed now in https://github.com/mozilla-mobile/fenix/issues/1682
🦄

**Pull request merged**